### PR TITLE
Feat: Quiz

### PR DESCRIPTION
**Pull request by pritam-kr**
Created new page
- Quiz page
- Quiz rule page
- Quiz result page

Screenshot (259)

Screenshot (260)

Screenshot (261)

Screenshot (262)

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]
Looks good to merge!

File: CSS/quiz.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+@import url(./rule.css);
+.question-title{
+    font-size: 1.6rem;
+}
+.q-list-wrapper{
```
Comment (srishti-maurya) [inline CSS/quiz.css:5]: maybe you may give a bit more descriptive name for better understanding like `ques-list-wrapper`

File: CSS/rule.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
+@import url(./CSS/common.css);
+@import url('/style.css');
+
+.rules{
+    margin: 2rem 0 2rem 2rem;
+}
+
+.rule-list{
```
Comment (srishti-maurya) [inline CSS/rule.css:8]: you may consistently use either of the name `rules` or `rule`

**Review by srishti-maurya — APPROVED** [review]
looks good.
liked the separation of sections in html files via comments for start and end.

File: CSS/rule.css (hunk 2 above)
Comment (pritam-kr) [reply CSS/rule.css:8]: I used rule for a single list item and rules for their parent element (ul).

File: CSS/quiz.css (hunk 1 above)
Comment (pritam-kr) [reply CSS/quiz.css:5]: ok

**Pull request merged**